lib/saito/ui/game-playerbox/game-playerbox.js

```diff
@@ -1,85 +1,147 @@
+ * This player has a status class added by default
+ * 
+ * Basic template: 
+ *   Name --- a Head for the Player's identicon and name
+ *   Info --- a Div for Information (formatted to the players specification)
+ *   Log --- a Div for Player log (to include additional information), 
+ *         this defaults to status for the player and is used by some games for interactive controls
+ *   Graphic --- a Div for graphical elements, such a dealt hand of cards
+ *
+ *   Name is the only mandatory and pre-defined part of Player-Box
+ *   You may insert whatever HTML you want into Info, Log, or Graphic. 
+ *   Info can be hidden/displayed independently of the Player-Box
+ *   Graphic can be assigned classnames for more flexible display behavior (such as outside the player-box)
+ */
 class GamePlayerBox {
 
+    /**
+     *  @constructor 
+     *  @param app - Saito app
+     */
     constructor(app) {
       this.app = app;
       this.game_mod = null;
     }
 
 
-    /*
-      Test is DOM contains element ID player-info-1, if not 
-      create all the player boxes, positioned by PlayersBoxArray 
+    /**
+     * Creates player boxes according to number of players in game
+     * Automatically assigns unique id #s to divs in DOM, but can be specified by setting a seats property in game object
+     * @param app - Saito app
+     * @param game_mod - the game object
     */
     render(app, game_mod) {
       this.game_mod = game_mod;
-      
-      try {
-        let playerboxes = [];
-         
+      try { 
         for (let i = 1; i <= game_mod.game.players.length; i++) {
           let player = this._playerBox(i);
-          if (!document.getElementById("player-box-"+player)) {
+          if (!document.getElementById(`player-box-${player}`)) {
+            //let isItMe = (this.returnPlayersBoxArray().indexOf(player) == 0); //Include status in box of the controlling player
             let pBox = app.browser.htmlToElement(GamePlayerBoxTemplate(player));
             if (document.querySelector(".main")) {
               document.querySelector(".main").append(pBox);
             } else { 
-	      document.body.append(pBox);
-	    }
+	            document.body.append(pBox);
+	          }
             this.refreshName(i); //Player names included in box by default
           }
-          if (document.getElementById(("player-box-"+player))) {
-            document.getElementById(("player-box-"+player)).style.display = "block";
+          if (document.getElementById((`player-box-${player}`))) {
+            document.getElementById((`player-box-${player}`)).style.display = "block";
           }
         }
       } catch (err) {console.log("Render error: ",err);}
     }
 
 
+    /** Doesn't do anything */
+    attachEvents(app, data=null) {
+        let chatmod = null;
+        let pb_self = this;
+          for (let i = 0; i < this.app.modules.mods.length; i++){
```

Comment:
Could (all) the `for`s be replace by simpler [forEach](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach#syntax)?